scipy/stats/_binned_statistic.py

```diff
@@ -501,8 +504,12 @@ def binned_statistic_dd(sample, values, statistic='mean',
     >>> with np.errstate(divide='ignore'):   # silence random axes3d warning
     ...     ax.bar3d(x, y, z, dx, dy, bincounts)
 
+    Reuse bin numbers and bin edges with new values:
+
+    >>> ret2 = stats.binned_statistic_dd(data, -np.arange(600), reuse=ret,
+    ...                                  statistic='mean')
     """
-    known_stats = ['mean', 'median', 'count', 'sum', 'std','min','max']
+    known_stats = ['mean', 'median', 'count', 'sum', 'std', 'min', 'max']
```

Comment:
nit: add a trailing comma e.g. `'max']` -> `'max',]`

Reply:
This makes flake8 unhappy.

Reply:
hmm, ok let's scratch that then. This ***is*** valid python but we've got flake8 configured to not have that in the codebase. If we've got that set that way no sense to change for only this line.